src/TailwindCssPreset.php

```diff
@@ -34,7 +34,7 @@ protected static function updatePackageArray(array $packages)
             'laravel-mix' => '^5.0.1',
             'postcss-import' => '^12.0',
             'postcss-nested' => '^4.2',
-            'tailwindcss' => '^1.4',
+            'tailwindcss' => '^1.7',
```

Comment:
```suggestion
            'tailwindcss' => '^1.8',
```

Reply:
> ```suggestion
> 
>             'tailwindcss' => '^1.8.2',
> 
> ```

Would be better since 1.8 was broken according to Adan Wathan, fix was added in 1.8.2 so it would be better to target version compatible with the fix, would prevent the broken 1.8 from being loaded.